src/Client.php

```diff
@@ -165,7 +301,31 @@ public static function make($rootCert, $clientCert, $baseUrl = self::SWISH_PRODU
             'verify' => $rootCert,
             'cert' => $clientCert,
             'handler' => HandlerStack::create(new CurlHandler()),
+            'base_uri' => $baseUrl
         ];
+
+        if(is_array($clientCert)) {
+            $passphrase = $clientCert[1];
+            $clientCert = $clientCert[0];
+        }
+        $certificateBody = file_get_contents($clientCert);
+        if ($certificateBody) {
+            $decodedCert = openssl_x509_parse($certificateBody, true);
```

Comment:
I'm assuming these x509-changes are here to make new Swish-certificates usable without modifications. That's probably a good idea. Add some comments to explain what you are doing. Also, we need to find a workaround for the statics.